documentation/modules/rn-2.6.adoc

```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-6_{context}"]
+=== Resolved issues 2.6.6
+
+.Runtime error: invalid memory address or nil pointer dereference
+
+In earlier releases of {project-short}, there was a runtime error of `invalid memory address` or `nil pointer dereference` caused by a pointer that was nil, and there was an attempt to access the value that it points to. This issue has been resolved in {project-short} 2.6.6. link:https://issues.redhat.com/browse/MTV-1353[(MTV-1353)]
+
+.All Plan and Migration pods scheduled to same node causing the node to crash
+
+In earlier releases of {project-short}, the scheduler could place all migration pods on a single node. When this happened, the node ran out of the resources. This issue has been resolved in {project-short} 2.6.6. link:https://issues.redhat.com/browse/MTV-1354[(MTV-1354)]
+
+.Empty bearer token is sufficient for authentication
```

Comment:
As this ticket is marked private, should we remove it from the release notes

Reply:
It's a CVE, how do we document CVEs?

Reply:
If you want me to include the CVE, then we can.

Personally, I would prefer to include them with a link to the Red Hat CVE library

| CVE example |
| ------------------- |
| 
![image](https://github.com/user-attachments/assets/7c88306b-bf04-462c-b2ff-70e78d4f6ac8) |

What do you think? If you want me to include it, please ping me the CVE number and I will include.

Reply:
Let's take what you prefer :)